src/entities/profile/api/types.ts

```diff
@@ -9,4 +14,18 @@ export type ProfileDetailsDto = ProfileDto & {
   email: string
   image_url: string,
   is_email_verified: boolean,
+  city: string,
+  is_private: boolean,
+  bio: string,
+  age: number,
+  date_of_birth: string,
+  category_favorite: IUserFavorite[] | []
+}
+   
+export type ProfileFollowingDto = {
```

Comment:
remove this type, it is not used in project.

*Use an `interface` instead of a `type`.eslint